src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header>      
+      <nav class="nav-bar">
+        <div class="logo">
+          <a href="#home"><img src="./images/logo.png" alt="MOYO logo"></a>
+        </div>
+        <div>
+          <a class="is-active nav-element" href="#apple">apple</a>
+          <a class="nav-element" href="#samsung">samsung</a>
+          <a class="nav-element" href="#smartphones">smartphones</a>
+          <a class="nav-element" data-qa="hover" href="#laptopsandcomputers">laptops & computers</a>
```

Comment:
```suggestion
          <a 
            class="nav-element" 
            data-qa="hover" 
            href="#laptops-and-computers"
          >
            laptops & computers
          </a>
```
Provide a new line for each attribute if their amount >= 3. 
Also, use kebab-case for href to make it readable